src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -497,7 +497,12 @@ file that was distributed with this source code.
 
                     // Some fields may be named with a dash (like keys of immutable array form type)
                     if (jQuery(defaultFieldId).length === 0) {
-                        return '#sonata-ba-field-container-{{ main_form_name }}-' + field;
+                        defaultFieldId = '#sonata-ba-field-container-{{ main_form_name }}-' + field;
+                    }
+
+                    // Some fields are rendered by form_widget (no container)
+                    if (jQuery(defaultFieldId).length === 0) {
```

Comment:
can you try to avoid jQuery on newer code? Maybe something like:

```
document.querySelector(defaultFieldId)
```

Reply:
Maybe not a big deal here since the same method is used two line before ?
We should rewrite all the method without jquery and not only part of it IMHO.

Reply:
I'll try to find some time and I remove jquery from this template.